core/tests/protractor_utils/forms.js

```diff
@@ -435,7 +435,7 @@ var expectRichText = function(elem) {
     // We select all top-level non-paragraph elements, as well as all children
     // of paragraph elements. (Note that it is possible for <p> elements to
     // surround, e.g., <i> tags, so we can't just ignore the <p> elements
-    // altogether.)
+    // altogether).
```

Comment:
I think the right place for the period is actually inside the parentheses since we're wrapping the whole sentence in parentheses. However, I think the parentheses don't really serve any purpose here, so we could also remove them.

Reply:
Done.